sale_operating_unit/models/sale.py

```diff
@@ -12,7 +12,7 @@ class SaleOrder(models.Model):
 
     @api.model
     def _default_operating_unit(self):
-        team = self.env['crm.team']._get_default_team_id()
+        team = self.env['crm.team'].sudo()._get_default_team_id()
```

Comment:
This sudo makes absolutely sense to me. The rest, no.

Reply:
what about self.team_id.sudo() ?
If we remove the sudos completely users will get an AccessError

Reply:
why? the sudo is for accessing methods, not values. sudo if for browse, search, etc.

Reply:
I don't know :see_no_evil:  

The thing is I got AccessError if don't use that. I don't see any other solution.

Reply:
Could you dive a bit deeper in the code while debugging to see the cause? :)

Reply:
The problem is the search in the sales team. Either we search using sudo, or we give permissions to the user.

Reply:
I prefer the sudo in the search